Doc: update tutorial about writting of custom PA modules (#369)
phracek/preupgrade-assistant: doc/module_writing_tutorial/README.md

| `05_add_pre_upgrade_script` | Simple template showing how to add a pre-upgrade script executed by redhat-upgrade-tool (RUT) before the prompt for reboot |
| `06_add_post_upgrade_script` | Simple template showing how to add a post-upgrade script to be executed automatically during post-upgrade phase (after the RPM upgrade transaction) |
| `07_module_in_python` | Simple example of a module written in Python in case you prefer this instead of bash |
| `08_install_custom_rpms` | Template of a module installing custom rpms located in a directory |

## Create a new PA module

We expect you want to just extend the IPU RHEL 6 -> 7 functionality. To create a new PA module from scratch, use the `preupgr-content-creator`
utility as described in the next section. To duplicate a template or an existing module and add it into the existing compose of PA modules, see the [Create a module without using the preupg-content-creator utility](#create-a-module-without-using-the-preupg-content-creator-utility) section.

### Create a module using the preupg-content-creator utility

> The simplest way to create an actor is using the `preupg-content-creator`

Reviewer: ```suggestion
The simplest way to create a module is using the `preupg-content-creator`
```
